bin/proxyctl

```
@@ -191,8 +191,7 @@ stop ()
 		# See https://github.com/docksal/service-vhost-proxy/issues/6 for more details on why this is necessary.
 		local network="${project_name}_default"
 		# '-I {}' and '{}' can be used to position the argument passed by xargs.
-		xargs -I {} docker network disconnect "$network" {} <<< "$project_containers"
-		docker network disconnect "$network" docksal-vhost-proxy
+		xargs -I {} docker network disconnect -f "$network" {} &>/dev/null <<< "$(docker ps -aq --no-trunc -f "network=${network}")"
```

We should not suppress errors unless we are 100% sure they are irrelevant.

```suggestion
		xargs -I {} docker network disconnect -f "$network" {} 1>/dev/null <<< "$(docker ps -aq --no-trunc -f "network=${network}")"
```